frontend/src/store/eam-group-configurate/actions.ts

```diff
@@ -24,6 +24,7 @@ const create = createAsyncThunk<
     name: registerPayload.name,
     tenantId: tenant?.id ?? '',
     workersIds: registerPayload.workersIds,
+    permissionIds: [], // will be fix on frontend branch
```

Comment:
```suggestion
    permissionIds: [],
```
you also need to handle frontend part in this task